.github/workflows/audit.yml

```diff
@@ -0,0 +1,15 @@
+name: Audit-Check
+on:
+  push:
+    paths:
+      - "**/Cargo.toml"
+      - "**/Cargo.lock"
+
+jobs:
+  security_audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/audit-check@v1
```

Comment:
Similar question regarding the cache: https://github.com/informalsystems/ibc-rs/blob/db92c7241ae6ea723bb4fbe1148de71f4f8a9493/.github/workflows/audit.yaml#L15-L21

But probably this could be done in a followup PR!